<ui version="4.0" >
 <author>Pere Constans</author>
 <class>findDialog</class>
 <widget class="QDialog" name="findDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find in text - cb2Bib</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkCase" >
        <property name="text" >
         <string>&amp;Case sensitive</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>180</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkWords" >
        <property name="text" >
         <string>&amp;Whole words only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Find:</string>
       </property>
       <property name="buddy" >
        <cstring>comboFind</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="c2bComboBox" name="comboFind" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearB" >
       <property name="toolTip" >
        <string>Clear Text</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="cb2bib.qrc" >
         <normaloff>:/icons/icons/clear_left.png</normaloff>:/icons/icons/clear_left.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="notFound" >
     <property name="text" >
      <string>Pattern not found</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Direction</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>147</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="radioBackward" >
        <property name="text" >
         <string>&amp;Backward</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radioForward" >
        <property name="text" >
         <string>&amp;Forward</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="Line" name="line5" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>c2bComboBox</class>
   <extends>QComboBox</extends>
   <header>c2bComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>clearB</tabstop>
  <tabstop>checkWords</tabstop>
  <tabstop>checkCase</tabstop>
  <tabstop>radioForward</tabstop>
  <tabstop>radioBackward</tabstop>
 </tabstops>
 <resources>
  <include location="cb2bib.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>clearB</sender>
   <signal>clicked()</signal>
   <receiver>comboFind</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>380</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel" >
     <x>244</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>findDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>findDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>200</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel" >
     <x>211</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
